compiler/test/test_end_to_end.ml

```diff
@@ -750,10 +754,10 @@ let tests =
   stdlib_tests @
   box_tests @
   loop_tests @
-  (*oom @ gc @*) 
+  (*oom @ *) gc @
```

Comment:
Maybe time to just get rid of `oom` if we aren't enabling them now?

Reply:
That was just an oversight, actually. They're now reenabled.